traveler_ui_kit/src/main/res/values/colors.xml

```diff
@@ -4,12 +4,14 @@
     <color name="colorPrimaryDark">#0074A1</color>
     <color name="colorAccent">#69BB48</color>
     <color name="colorSecondary">#FC9B44</color>
+    <color name="glxOrange">#EA943E</color>
```

Comment:
Why are there custom colours on the SDK side?

Reply:
removed